tests/backup/backup_nfs_test.go

```diff
@@ -203,3 +205,222 @@ var _ = Describe("{DeleteNfsExecutorPodWhileBackupAndRestoreInProgress}", func()
+			appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, appNamespaces)
+			err = CreateBackupWithValidation(ctx, multipleNamespaceBackup, SourceClusterName, bkpLocationName, backupLocationUID, appContextsToBackup, labelSelectors, orgID, clusterUid, "", "", "", "")
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup [%s]", multipleNamespaceBackup))
+			backupNames = append(backupNames, multipleNamespaceBackup)
+		})
+
+		Step("Taking custom resource backup of all namespaces", func() {
+			log.InfoD("Taking custom resource backup of all namespaces")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			customResourceBackupName = fmt.Sprintf("%s-custom-resource-%v", BackupNamePrefix, RandomString(5))
+			appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, appNamespaces)
+			err = CreateBackupWithCustomResourceTypeWithValidation(ctx, customResourceBackupName, SourceClusterName, bkpLocationName, backupLocationUID, appContextsToBackup, []string{"PersistentVolumeClaim"}, nil, orgID, clusterUid, "", "", "", "")
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creation of custom resource backup - %s", customResourceBackupName))
+			backupNames = append(backupNames, customResourceBackupName)
+		})
+
+		Step("Remove the JSON files from the NFS backup location for all the backups", func() {
+			log.InfoD("Remove the JSON files from the NFS backup location for all the backups")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				backupUID, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+				log.FailOnError(err, fmt.Sprintf("Getting UID for backup %v", backupName))
+				backupInspectRequest := &api.BackupInspectRequest{
+					Name:  backupName,
+					Uid:   backupUID,
+					OrgId: orgID,
+				}
+				resp, err := Inst().Backup.InspectBackup(ctx, backupInspectRequest)
+				log.FailOnError(err, fmt.Sprintf("error inspecting backup %v", backupName))
+				currentBackupPath := globalBucket + "/" + resp.Backup.BackupPath
+				log.Infof("Deleting the JSON files from the NFS backup location for backup %v", backupName)
+				DeleteJSONFilesFromNFSLocation(currentBackupPath)
+			}
+		})
+
+		Step("Verify if the backups are in CloudBackupMissing state after the JSON file deletion", func() {
+			log.InfoD("Verify if the backups are in CloudBackupMissing state after the JSON file deletion")
+			var wg sync.WaitGroup
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				wg.Add(1)
+				go func(backupName string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					bkpUid, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+					log.FailOnError(err, "Fetching backup uid")
```

Comment:
include backupName?

Reply:
Addressed

Reply:
Is backup name included in the log message?